resources/views/individuals/edit/about-you.blade.php

```diff
@@ -12,11 +12,13 @@
                 {{ __('Step :current of :total', ['current' => request()->get('step') ?? 1, 'total' => $individual->isConnector() ? 5 : 4]) }}<br />
                 {{ __('About you') }}
             </h2>
+            <x-interpretation class="interpretation--start" name="{{ __('About you', [], 'en') }}" />
 
             <hr class="divider--thick">
 
             <div class="field @error('name') field--error @enderror">
                 <x-hearth-label for="name"><x-required>{{ __('Name') }}</x-required></x-hearth-label>
+                <x-interpretation class="interpretation--start" name="{{ __('Name', [], 'en') }}" />
```

Comment:
Thanks @jobara, I was thinking, if an interpretation is for a form field, wouldn't it have either "required" or "optional" in the name? If that's the case, I think we can have two different instances, for example "Name (required)" and "Name".

Reply:
I left a comment about this in my upcoming review, but I just remembered that the interpretation name doesn't get added to the localization files because it is dynamically used. So these unique names won't be localized. It isn't a huge issue at the moment because the admin panels aren't localized, but once they are these will break. I'm thinking we should make this info to the namespace instead, but we talk it through.